metabase/metabase
# chore: rework DataSelector styles to css modules

[epic](https://github.com/metabase/metabase/issues/50983)

### Description

PR 3/?

Rework DataSelector to css modules

### How to verify
verify pickers. e.g. follow steps from this [PR](https://github.com/metabase/metabase/pull/50535)

<img width="1005" alt="Screenshot 2024-12-05 at 17 18 18" src="https://github.com/user-attachments/assets/c88cfe2d-6087-49a7-8db9-0cd9166ac88c">
